### DISCUSSION TITLE
Lost connection to NHC

### DISCUSSION THREAD

**Discussion by whitespacecode** [joleys/niko-home-control-II#197] [Q&A] [answered]
My NHC changes ip address because of internet failure, power failure,..  
There is nothing in the notifications that notify this.

So i was thinking it would be time to set a fixed ip to my NHC.
Do you know where and how i can do that?

If so, maybe you could update the docs on how to set a fixed ip on the NHC

**Comment by tijsverkoyen** [comment, marked as answer]
I have done this in my DHCP server.
But I think you can do this in the Niko Home Control Programming software.
